crates/uv-resolver/src/resolution_mode.rs

```diff
@@ -20,6 +20,16 @@ pub enum ResolutionMode {
     LowestDirect,
 }
 
+impl std::fmt::Display for ResolutionMode {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Self::Highest => write!(f, "highest"),
+            Self::Lowest => write!(f, "lowest"),
+            Self::LowestDirect => write!(f, "lowest-direct"),
```

Comment:
Does anyone know how to get these automatically from the Serde implementation, in `to_toml`?